hw/top_earlgrey/sw/autogen/top_earlgrey.h

```diff
@@ -1775,8 +1793,8 @@ typedef enum top_earlgrey_hintable_clocks {
  * configuration space, i.e. ROM, main SRAM, and flash are excluded but
  * retention SRAM, spi_device memory, or usbdev memory are included.
  */
-#define TOP_EARLGREY_MMIO_BASE_ADDR 0x40000000u
-#define TOP_EARLGREY_MMIO_SIZE_BYTES 0x10000000u
+#define TOP_EARLGREY_MMIO_BASE_ADDR 0x0u
```

Comment:
That change is because the DMI interface is not mapped in Earlgrey. That change will be reverted once https://github.com/lowRISC/opentitan/pull/24718 gets merged. This PR allows the generically add devices/interfaces to the MMIO region. For Earlgrey, the DMI won't be added and thus also not considered for the MMIO region computation.

Reply:
This is fixed on master.